### PR TITLE
TS-4249: Deprecate stale_while_revalidate plugin.

### PR DESCRIPTION
**Pull request by PSUdaemon**

### CODE REVIEW

**Comment by atsci** [comment]
FreeBSD build *successful*! See https://ci.trafficserver.apache.org/job/Github-FreeBSD/576/ for details.

**Comment by atsci** [comment]
Linux build *successful*! See https://ci.trafficserver.apache.org/job/Github-Linux/472/ for details.

**Comment by bryancall** [comment]
👍 - Looks good

File: plugins/experimental/stale_while_revalidate/README [hunk 1]
```diff
@@ -1,3 +1,13 @@
+************************************************************
+    THIS PLUGIN IS DEPRECATED IN FAVOR OF CHANGE TO CORE
+************************************************************
+
+You can still run `make && make install` in this directory
```
Comment (jpeach) [inline plugins/experimental/stale_while_revalidate/README]: This won't work ... ``automake`` won't generate the Makefile.

Comment (PSUdaemon) [inline plugins/experimental/stale_while_revalidate/README]: It will since it's still in `configure.ac`. It just won't build with a top level `make`.

Comment (jpeach) [inline plugins/experimental/stale_while_revalidate/README]: Ok 👍  then

**Comment by zwoop** [comment]
Yeh, I like this, major drag we have to keep it for another full year. :-/.

:+1:

**Pull request merged**